From deed29a6cc3797be1c19de88cd3adcb37413cf04 Mon Sep 17 00:00:00 2001
From: Thilo Graf <dbt@novatux.de>
Date: Wed, 23 Dec 2015 23:22:20 +0100
Subject: [PATCH 19/21] CPersonalizeGui: try to fix wrong active mode after
 changed personalize setup

Item must be enabled here, because if member values active and current_active
are different, item is handled as disabled!
This effect was observed for radio or/and tv mode switches. Both items
were disabled after applied personalize settings. Neutrino restart was required
for correct item modes.
---
 src/gui/personalize.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gui/personalize.cpp b/src/gui/personalize.cpp
index 0e82548..fe5f63d 100644
--- a/src/gui/personalize.cpp
+++ b/src/gui/personalize.cpp
@@ -983,6 +983,7 @@ void CPersonalizeGui::addPersonalizedItems()
 				{
 					//add item
 					v_item[i].widget->addItem(v_item[i].menuItem, v_item[i].default_selected); //forwarders...
+					v_item[i].menuItem->current_active = true; //item must be enabled here, because if members active and current_active are different, item is handled as disabled!
 					allow_sep = true;
 
 					//generate shortcut for next item
-- 
2.1.4

